packages/compile-service/src/index.ts

```diff
@@ -43,17 +42,35 @@ export type DbDeployQueue = {
     await proofDb.connect();
   }
 
+  const modelTransaction = ModelDbTransaction.getInstance();
+  const modelDbSettings = ModelDbSetting.getInstance();
+
   while (true) {
     const request = await redisQueue.dequeue();
     if (request) {
-      logger.info(`Received ${request.databaseName} to queue`);
+      const tx = await modelTransaction.findById(request.id);
+
+      if (!tx) {
+        logger.error(`Transaction ${request.id} has not been found`);
```

Comment:
Dont forget to enqueue with the id, I've comment in the enqueue code, when I tested it show undefined because you forgot to put it

Reply:
done